client/my-sites/checkout/composite-checkout/lib/filter-appropriate-payment-methods.ts

```diff
@@ -97,9 +112,18 @@ function isPaymentMethodEnabled(
 	method: CheckoutPaymentMethodSlug,
 	allowedPaymentMethods: null | CheckoutPaymentMethodSlug[]
 ): boolean {
+	// Redirect payments might not be possible in some cases - for example in the desktop app
+	if (
+		redirectPaymentMethods.includes( method ) &&
```

Comment:
Combined with the above comment, here we could say `if ( isRedirectPaymentMethod( method ) ... `